test/tool/resources/concat.json

```diff
@@ -0,0 +1,303 @@
+{
+  "tests": [
```

Comment:
This tests part is reused the [tests part](https://github.com/BruceDai/wpt/blob/add_webnn_concat_tests/webnn/resources/test_data/concat.json#L2) of test_data file for WPT WebNN concat tests.